composer.json

```diff
@@ -11,37 +11,37 @@
     },
     "require" : {
         "php" : ">=7.1.0",
-        "adldap2/adldap2" : "9.1.3",
-        "adodb/adodb-php" : "5.20.12",
+        "adldap2/adldap2" : "10.0.8",
```

Comment:
@sharonco-matrix ,
Just wanted to warn your group of this upgrade in the adldap2 package. Doesn't appear to be any big changes, but would be great to hear that this is testing well for your group. thanks! -brady